### DISCUSSION TITLE
This tool has moved to UMEP for processing, available via the QGIS Plugin Manager. Visit our online manual for more information.

### DISCUSSION THREAD

**Discussion by kun-creator** [UMEP-dev/UMEP#699] [General]
![Image](https://github.com/user-attachments/assets/fb4e1a46-e686-42a3-bc95-ce24e8e35f41)
I've already installed UMEP for processing.But when I tried to use Sky View Factor,it was wrong.

**Comment by biglimp** [comment]
Yes, so you need to open the tool from the Processing toolbox in QGIS.

**Comment by wangguo-ops** [comment]
![1736860699626](https://github.com/user-attachments/assets/255bda31-143d-426a-bbb2-b381528be0b0)
